integration.py

```diff
@@ -322,7 +322,8 @@ def _run_report(rev):
             return output_path
 
         Environment.checkout(rev)
-        Environment.sync()
+        # NOTE: this sync only copies schemas and uses cached data
+        Environment.sync({"INCLUDE_DATA": "false"})
```

Comment:
Can you clarify a bit on why this change was required?

Reply:
The SRC_DATA_BUCKET and SRC_DATA_PREFIX aren't being set in the arguments, so it'll error when it gets used in the bash script.